zend_abstract_interface/json/json.h

```diff
@@ -49,7 +49,11 @@ static inline int zai_json_decode_assoc(zval *return_value, const char *str, int
 }
 #endif
 
+#ifdef __APPLE__
+extern __attribute__((weak)) __attribute__((weak_import)) zend_class_entry *php_json_serializable_ce;
```

Comment:
We should be able to use this shorter syntax, FYI:

```suggestion
extern __attribute__((weak, weak_import)) zend_class_entry *php_json_serializable_ce;
```